pkg/reconciler/apis/apibinding/apibinding_reconcile_test.go

```diff
@@ -579,7 +580,7 @@ func TestReconcileBinding(t *testing.T) {
 					Status:   corev1.ConditionFalse,
 					Severity: conditionsv1alpha1.ConditionSeverityError,
 					Reason:   apisv1alpha1.NamingConflictsReason,
-					Message:  "naming conflict with APIBinding conflicting",
+					Message:  "naming conflict with a bound CRD conflicting, spec.names.plural=widgets is forbidden",
```

Comment:
the concept of a bound CRD should not exposed to the user. Would call it a bound API.